<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>524</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>8</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>11</y>
     <width>791</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QFrame" name="frameSerial">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QSlider" name="horizontalSlider">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>90</y>
         <width>361</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1000</number>
       </property>
       <property name="maximum">
        <number>2000</number>
       </property>
       <property name="sliderPosition">
        <number>1000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>330</y>
         <width>231</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>60</y>
         <width>121</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>MSP Motor Control</string>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonPassthough">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>10</y>
         <width>121</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Passthrough Start</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>420</y>
         <width>361</width>
         <height>61</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>400</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Console</string>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonSendMessage">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>330</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>send</string>
       </property>
      </widget>
      <widget class="QComboBox" name="serialSelectorBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="currentText">
        <string>Serial Port</string>
       </property>
       <item>
        <property name="text">
         <string>Serial Port</string>
        </property>
       </item>
      </widget>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>400</y>
         <width>81</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Hex Format</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>264</width>
         <height>91</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>231</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Serial Port Info</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>310</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Send Message</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>60</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonConnect">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>10</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>open</string>
       </property>
      </widget>
      <widget class="QLabel" name="StatusLabel">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>40</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Status:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QFrame" name="fourWayFrame">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>110</y>
         <width>411</width>
         <height>381</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="initMotor4">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>10</y>
          <width>31</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>M4</string>
        </property>
       </widget>
       <widget class="QPushButton" name="initMotor3">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>10</y>
          <width>31</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>M3</string>
        </property>
       </widget>
       <widget class="QPushButton" name="initMotor1">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>10</y>
          <width>31</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>M1</string>
        </property>
       </widget>
       <widget class="QLabel" name="connectMotor">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Connect and Read Settings</string>
        </property>
       </widget>
       <widget class="QPushButton" name="initMotor2">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>10</y>
          <width>31</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>M2</string>
        </property>
       </widget>
       <widget class="QFrame" name="settingsFrame">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>391</width>
          <height>331</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="writeBinary">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>300</y>
           <width>101</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>flash write</string>
         </property>
        </widget>
        <widget class="QPushButton" name="loadBinary">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>300</y>
           <width>101</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>load binary</string>
         </property>
        </widget>
        <widget class="QPushButton" name="VerifyFlash">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>300</y>
           <width>101</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>flash verify</string>
         </property>
        </widget>
        <widget class="QFrame" name="eepromFrame">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>50</y>
           <width>381</width>
           <height>201</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QCheckBox" name="rvCheckBox">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>121</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Reverse Rotation</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="biDirectionCheckbox">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>131</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Bi-directional (fwd/rv)</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="sinCheckBox">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>131</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Sinusoidal Startup</string>
          </property>
         </widget>
         <widget class="QPushButton" name="writeEEPROM">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>170</y>
            <width>151</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Write EEprom Settings</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="comp_pwmCheckbox">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>100</y>
            <width>161</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Complementary PWM</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="varPWMCheckBox">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>130</y>
            <width>121</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Variable PWM</string>
          </property>
         </widget>
         <widget class="QSlider" name="horizontalSlider_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>10</y>
            <width>81</width>
            <height>22</height>
           </rect>
          </property>
          <property name="maximum">
           <number>8</number>
          </property>
          <property name="pageStep">
           <number>2</number>
          </property>
          <property name="value">
           <number>4</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QSlider" name="horizontalSlider_3">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>40</y>
            <width>81</width>
            <height>22</height>
           </rect>
          </property>
          <property name="value">
           <number>99</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QLabel" name="timingLabel">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>10</y>
            <width>101</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>TO DO</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>40</y>
            <width>91</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>TO DO</string>
          </property>
         </widget>
         <widget class="QSlider" name="horizontalSlider_4">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>70</y>
            <width>81</width>
            <height>22</height>
           </rect>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>4</number>
          </property>
          <property name="value">
           <number>2</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>70</y>
            <width>81</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>TO DO</string>
          </property>
         </widget>
         <zorder>rvCheckBox</zorder>
         <zorder>biDirectionCheckbox</zorder>
         <zorder>sinCheckBox</zorder>
         <zorder>comp_pwmCheckbox</zorder>
         <zorder>varPWMCheckBox</zorder>
         <zorder>horizontalSlider_2</zorder>
         <zorder>horizontalSlider_3</zorder>
         <zorder>timingLabel</zorder>
         <zorder>label_5</zorder>
         <zorder>horizontalSlider_4</zorder>
         <zorder>label_4</zorder>
         <zorder>writeEEPROM</zorder>
        </widget>
        <widget class="QPushButton" name="sendFirstEEPROM">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>20</y>
           <width>131</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Send Default Settings</string>
         </property>
        </widget>
        <widget class="QLabel" name="escStatusLabel">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>241</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>ESC Status : </string>
         </property>
        </widget>
        <widget class="QProgressBar" name="progressBar">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>260</y>
           <width>371</width>
           <height>23</height>
          </rect>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QCheckBox" name="devSettings">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>121</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Show Dev Controls</string>
       </property>
      </widget>
      <widget class="QFrame" name="devFrame">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>311</width>
         <height>101</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QComboBox" name="FourWayComboBox">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>40</y>
          <width>121</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Test_Alive_30</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Get_Version_31</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Get_IF_Name_32</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Reset_ESCs_35</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Device_Init_flash_37</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Device_Read(3a)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>device_write(3b)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Set_interface(Arm)_3F</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Set_interface(Silabs)_3F</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Exit_Interface_34</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CheckifCrcWorking</string>
         </property>
        </item>
       </widget>
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>10</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>104</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="sendMspComButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>131</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Send MSP Command</string>
        </property>
       </widget>
       <widget class="QPushButton" name="fourWaySendButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>141</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Send 4 Way Command</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>InitDevice(37)</string>
        </property>
       </widget>
       <widget class="QPushButton" name="buttonSetIfARM">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>70</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>SetIF(ARM)</string>
        </property>
       </widget>
      </widget>
      <zorder>fourWayFrame</zorder>
      <zorder>horizontalSlider</zorder>
      <zorder>plainTextEdit</zorder>
      <zorder>checkBox</zorder>
      <zorder>buttonPassthough</zorder>
      <zorder>plainTextEdit_2</zorder>
      <zorder>label_3</zorder>
      <zorder>buttonSendMessage</zorder>
      <zorder>serialSelectorBox</zorder>
      <zorder>checkBox_2</zorder>
      <zorder>textEdit</zorder>
      <zorder>label_2</zorder>
      <zorder>label</zorder>
      <zorder>lineEdit</zorder>
      <zorder>buttonConnect</zorder>
      <zorder>StatusLabel</zorder>
      <zorder>devSettings</zorder>
      <zorder>devFrame</zorder>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
